<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TOTAL_ThresholdDlg</class>
 <widget class="QDialog" name="TOTAL_ThresholdDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="setThresholdBtn">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>190</y>
     <width>120</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>修改阀值</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quitBtn">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>190</y>
     <width>120</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveBtn">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>190</y>
     <width>120</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="refreshBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>120</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>刷新</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>581</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>300</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>总电流阀值</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tableWidget</tabstop>
  <tabstop>quitBtn</tabstop>
  <tabstop>saveBtn</tabstop>
  <tabstop>refreshBtn</tabstop>
  <tabstop>setThresholdBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
